src/development/dev-git-branching-strategy.md

```diff
@@ -98,14 +109,22 @@ Git repository hosting providers provide various features to add additional cont
   * Required approvals at `1` as a minimum. Projects may choose more approvers.
 * Under `Settings` > `Merge requests`
   * Merge method: `Fast-forward merge`
+* Under `Settings` > `Push rules` > `Branch name`
+  * Branch name regex: `(hotfix|develop|build|ci|docs|feature|fix|perf|refactor|style|test)[\-/]([A-Z]{3,4}|NO-JIRA)`
```

Comment:
this wont allow branch such as main or develop-wcag-changes. `(develop|main)|(hotfix|develop|build|ci|docs|feature|fix|perf|refactor|style|test)[\-\/]([A-Z]{3,4}|NO-JIRA)`

Reply:
fixed